### PR TITLE
Add Jetpack powered badge to Me and App Settings

### PR DESCRIPTION
**Pull request by Gio2018**
Fixes #NA

This PR adds the "Jetpack powered" badge to the Me and App Settings screens (see screenshots).
It also fixes the distance of the badge from the top in activity detail, to align it with the designs and the other badges

Me iPad Dark | App Settings iPhone Light
-- | --
 | 

To test:

- Build/run the WordPress target on this branch
- Go to Me and make sure the "Jetpack powered" badge appears at the bottom
- Tap App Settings and make sure the "Jetpack powered" badge appears at the bottom
- Stop execution and build the Jetpack target
- Make sure the banner does not appear in both cases listed above
- Stop the execution, turn the `jetpackPowered` feature flag to false
- Re-build/run the WordPress target and make sure the banner does not appear on the same cases

## Regression Notes
1. Potential unintended areas of impact
Low, adds a row to both Me and App Settings

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test on several devices

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] ~~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~~ Not yet released.

### CODE REVIEW

**Review by twstokes — COMMENTED** [review]
👋 @Gio2018, the code looks good to me. I noticed that the "Log Out" cell is squared off on the bottom - should we round that out?

**Comment by Gio2018** [comment]
>  I noticed that the "Log Out" cell is squared off on the bottom - should we round that out?

Whoops!! Thanks for spotting that @twstokes ! The resulting code is also simpler 🤦 . Should be fixed now.

**Review by twstokes — APPROVED** [review]
LGTM @Gio2018 🚀 

I merged `trunk` in and tested:
- Landscape orientation
- Dark mode
- Large a11y sizes

File: WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift [hunk 1]
```diff
@@ -37,7 +37,7 @@ class MeViewController: UITableViewController {
             NavigationItemRow.self,
             IndicatorNavigationItemRow.self,
             ButtonRow.self,
-            DestructiveButtonRow.self
+            DestructiveButtonRow.self,
```
Comment (sla8c) [inline WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift]: nit - did you intend to have this trailing comma for the last element in this Array?

**Review by sla8c — APPROVED** [review]
I left one nit comment but otherwise LGTM

I tested on:
 - iPad portrait and landscape
 - iPhone portrait and landscape
 - dark mode on both of the above

File: WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift (hunk 1 above)
Comment (Gio2018) [reply WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift]: I did not, thanks for catching it!

**Comment by Gio2018** [comment]
Thank you for reviewing @twstokes @sla8c !!

**Pull request merged**